pkg/kapp/app/recorded_app.go

```diff
@@ -80,9 +85,26 @@ func (a *RecordedApp) UpdateUsedGVs(gvs []schema.GroupVersion) error {
 func (a *RecordedApp) CreateOrUpdate(labels map[string]string) error {
 	defer a.logger.DebugFunc("CreateOrUpdate").Finish()
 
+	if strings.ToLower(os.Getenv("USE_OLD_CONFIGMAP_NAME")) == "true" {
+		return a.createOrUpdate(a.name, labels)
+	}
+
+	configMap, err := a.coreClient.CoreV1().ConfigMaps(a.nsName).Get(context.TODO(), a.name, metav1.GetOptions{})
+	if err == nil {
+		if _, ok := configMap.Labels[KappIsAppLabelKey]; !ok {
+			return fmt.Errorf("Unable to migrate, configmap '%s' is not marked as a kapp app", a.name)
```

Comment:
I think we can proceed on to creating the app with suffix in this case.

Reply:
Right!